src/app/fdctl/run/tiles/fd_pack.c

```diff
@@ -166,28 +177,32 @@ before_credit( void * _ctx,
     fd_pack_insert_txn_cancel( ctx->pack, ctx->cur_spot );
     ctx->cur_spot = NULL;
   }
+}
+
+static inline void
+after_credit( void *             _ctx,
+              fd_mux_context_t * mux ) {
+  fd_pack_ctx_t * ctx = (fd_pack_ctx_t *)_ctx;
 
   /* If we time out on our slot, then stop being leader. */
   long now = fd_log_wallclock();
   if( FD_UNLIKELY( now>=ctx->slot_end_ns && ctx->leader_slot!=ULONG_MAX ) ) {
     ctx->leader_slot = ULONG_MAX;
     fd_pack_end_block( ctx->pack );
+    fd_mux_publish( mux, fd_disco_poh_sig( ctx->slot_microblock_cnt, POH_PKT_TYPE_DONE_PACKING, ULONG_MAX ), 0UL, 0UL, 0UL, 0UL, 0UL );
```

Comment:
It's worth a comment somewhere that you're sticking microblock cnt in the slot field